src/main/kotlin/org/wycliffeassociates/otter/jvm/recorder/app/view/drawables/WaveformLayer.kt

```diff
@@ -0,0 +1,35 @@
+package org.wycliffeassociates.otter.jvm.recorder.app.view.drawables
+
+import javafx.scene.canvas.Canvas
+import javafx.scene.canvas.GraphicsContext
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.common.app.ActiveRecordingRenderer
+
+class WaveformLayer(private val renderer: ActiveRecordingRenderer) :
+        Drawable
```

Comment:
can you put this on the previous line and get rid of the empty line two lines lower? Or did you do this to draw attention to the interface?

Reply:
I may have had other parameters or other interfaces and did it to not make the line too long